src/components/map/CuriosityMap.tsx

```diff
@@ -1,9 +1,26 @@
-import React from "react";
+import React, {useState} from "react";
+import './Map.scss'
```

Comment:
Semicolon and double quotes please!
```suggestion
import "./Map.scss";
```